src/phantom.js

```diff
@@ -27,25 +28,40 @@ export default class Phantom {
      * Creates a new instance of Phantom
      *
      * @param args command args to pass to phantom process
-     * @param [config] configuration object
-     * @param [config.phantomPath] path to phantomjs executable
+     * @param [phantomPath] path to phantomjs executable
+     * @param [logger] object containing functions used for logging
+     * @param [logLevel] log level to apply on the logger (if unset or default)
      */
-    constructor(args = [], config = {}) {
+    constructor(args = [], {phantomPath = phantomjs.path, logger = defaultLogger, logLevel = defaultLogLevel} = {
```

Comment:
This was a funky one. But essentially the left side sets default values per key and the right sets the default object if it is absent.